### PR TITLE
End point posting and getting proverbs

### PR DESCRIPTION
**Pull request by AlsasaAmmar**

### CODE REVIEW

File: yarn-error.log [hunk 1]
```diff
@@ -0,0 +1,57 @@
+Arguments:
```
Comment (frankPairs) [inline yarn-error.log:1]: you don't need to push this file to the repository. it is a good practice to add it to the .gitignore file.

File: controllers/proverbs-controllers.js [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+
+const getProverbs = async (req, res, next) => {
+	let proverbs;
+	try {
+		proverbs = await Proverb.find({});
+	} catch (error) {
+		console.log(error);
+		res.status(500).json({
+			msg: 'could not retrieve proverbs from database'
+		});
```
Comment (frankPairs) [inline controllers/proverbs-controllers.js:15]: here is messing the return next(error); line

File: controllers/proverbs-controllers.js [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+			msg: 'Could not find user in database'
+		});
+		return next(error);
+	}
+
+	if (!user) {
+		res.status(500).json({
+			msg: 'Could not find user'
+		});
+		throw new Error('Invalid credentials.');
```
Comment (frankPairs) [inline controllers/proverbs-controllers.js:70]: here you could follow the same convention to trigger errors using next:

```
return next(new Error('Invalid credentials.'));
```

**Review by frankPairs — COMMENTED** [review]
good job!

I added some comments related to this PR.

File: yarn-error.log (hunk 1 above)
Comment (sarah-vanderlaan) [reply yarn-error.log:1]: @AlsasaAmmar I see that you resolved this conversation but this file is still in the PR - can you remove it? :)

Comment (AlsasaAmmar) [reply yarn-error.log:1]: I added it to the gitignore. Should I delete it from here?

Comment (sarah-vanderlaan) [reply yarn-error.log:1]: yesss :) you just want to have this file locally, not in the repository

**Pull request merged**